Queuesim/testTrafficSource.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+import simpy
+import numpy as np
+
+from TrafficSource import TrafficSourceConstInterarrival
+from Server import ServerConstServiceTime
+
+class testTrafficSource(unittest.TestCase):
+
+    def setUp(self):
+        service_time = 5
+        self.__env__ = simpy.Environment()
+        resource = simpy.Resource(self.__env__, capacity=1)
+        self.__server__ = ServerConstServiceTime(self.__env__, resource, service_time)
+
+    def test_single_source(self):
+        interarrival_time = 2
+        number_of_packets = 5
+        source = TrafficSourceConstInterarrival(self.__env__, self.__server__, interarrival_time)
+        source.add_traffic_generator_process(number_of_packets)
+        self.__env__.run()
+        response_times = [x.response_time() for x in source.traffic()]
+
+        np.testing.assert_almost_equal(response_times, [5, 8, 11, 14, 17])
```

Comment:
Please do not use magic numbers. Better to declare a variable something like:
expected_response_times = [5, 8, 11, 14, 17]